notebooks/parse_tfrecords.ipynb

```diff
@@ -1,168 +1,250 @@
 {
```

Comment:
<p>Didn't we have to apply transposition trick to make it work? I remember we ran into errors without it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/semantic-segmentation-ml-pipeline/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I think we needed it to let TF know the shape of the Tensors, so I will apply `reshape` in the preprocessing step

Reply:
Okay. Cool.